contracts/src/storage/AssetsStorage.sol

```diff
@@ -6,7 +6,6 @@ import {TokenInfo, ParaID} from "../Types.sol";
 
 library AssetsStorage {
     struct Layout {
-        mapping(address token => TokenInfo) tokenRegistry;
```

Comment:
oops, we can't move storage fields, otherwise the upgraded gateway's code will be confused about the storage layout.